.travis.yml

```diff
@@ -2,20 +2,24 @@ language: java
 addons:
   apt:
     sources:
-    - r-packages-precise
+      - r-packages-precise
     packages:
-    - r-base
+      - r-base
 jdk:
-- oraclejdk8
+  - oraclejdk8
 env:
   matrix:
-  - TERM=dumb
+    - TERM=dumb
+    - TERM=dumb DATAFLOW_RUNNER=SparkPipelineRunner
```

Comment:
I was looking at the travis docs, and I'm not sure if this should be repeated here and in the allow failures section.  Do you know for sure?  Could you pull allow_failures subsection up into this matrix block so that the two together as well.  

http://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix